app/audit/models/ChargeSummaryAudit.scala

```diff
@@ -98,7 +98,7 @@ case class ChargeSummaryAudit(mtdItUser: MtdItUser[_], docDateDetail: DocumentDe
   }
 
   private val interestPeriod: Option[String] = (docDateDetail.documentDetail.interestFromDate, docDateDetail.documentDetail.interestEndDate) match {
-    case (Some(fromDate), Some(endDate)) => Some(fromDate + " to " + endDate)
+    case (Some(fromDate), Some(endDate)) => Some(fromDate.toString + " to " + endDate.toString)
```

Comment:
string interpolation will be shorted in this case, ie:
Some(s"$fromDate to $endDate")